tokio/src/process/unix/orphan.rs

```diff
@@ -1,6 +1,9 @@
+use crate::signal::unix::driver::Handle as SignalHandle;
+use crate::signal::unix::{signal_with_handle, SignalKind};
+use crate::sync::watch;
 use std::io;
 use std::process::ExitStatus;
-use std::sync::Mutex;
+use std::sync::{Mutex, MutexGuard};
```

Comment:
Shouldn't this be using the `parking_lot` mutex when that feature is enabled? You do that by using the following mocked import, which changes between the std and `parking_lot` mutex depending on the features.
```suggestion
use crate::loom::sync::{Mutex, MutexGuard};
```

Reply:
Good catch! I haven't done much work with loom so this slipped my mind